### PR TITLE
4894 email reminders add notification setting for make decision reminders

### PR DESCRIPTION
**Pull request by SamJamCul**
## Context
https://trello.com/c/V1nCbdYC/4894-email-reminders-add-notification-setting-for-make-decision-reminders
Adding a column to the `provider_user_notifications` for the make decision email reminder

## Changes proposed in this pull request

image

- [x] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -597,8 +599,8 @@
   create_table "provider_users_providers", force: :cascade do |t|
     t.bigint "provider_id", null: false
     t.bigint "provider_user_id", null: false
-    t.datetime "created_at", default: -> { "now()" }, null: false
-    t.datetime "updated_at", default: -> { "now()" }, null: false
+    t.datetime "created_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "updated_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
```
Comment (SamJamCul) [inline db/schema.rb:603]: Not totally sure where this has come from - anyone have any ideas?

**Review by cwrw — APPROVED** [review]
looks good! Just note the rebase. and making sure we don't have the feature flag commit

**Review by cwrw — COMMENTED** [review]
Ah actually can you add this new column to `config/analytics.yml` to make sure we send it to big query when its populated?

**Review by cwrw — APPROVED** [review]
💯

**Pull request merged**